en/04_Changelogs/4.13.0.md

```diff
@@ -76,3 +76,4 @@ If you are using this release as a step towards upgrading to Silverstipe CMS 5,
 ## Features and enhancements
 
 - There is a new [`Environment::hasEnv()`](api:SilverStripe\Core\Environment::hasEnv()) method which will tell you if a given environment variable has been set. This is useful because [`Environment::getEnv()`](api:SilverStripe\Core\Environment::getEnv()) will return `false` if the variable hasn't been set, but it will also return `false` if the variable has been explicitly set to `false`! Now you can check which it is.
+- For `silverstripe/graphql` v4 an additional fallback to rebuild the schema is a file is missing has been added. There is a default configurable minimum interval between this type of rebuild occuring of 15 minutes. This type of rebuild can also be disabled via configuration. These configuration options are available on [`SilverStripe\GraphQL\Schema\Storage\AbstractTypeRegistry`](api:SilverStripe\GraphQL\Schema\Storage\AbstractTypeRegistry).
```

Comment:
```suggestion
- For `silverstripe/graphql` v4 an additional fallback has been added  to rebuild the schema if a file is missing. There is a configurable minimum interval between this type of rebuild occuring, which is set to 15 minutes by default. This type of rebuild can also be disabled via configuration. These configuration options are available on [`SilverStripe\GraphQL\Schema\Storage\AbstractTypeRegistry`](api:SilverStripe\GraphQL\Schema\Storage\AbstractTypeRegistry).
```

We also normally have examples of setting the config in yaml code blocks for this sort of thing - I wonder if it's worth giving this it's own heading?

Reply:
Updated

Reply:
The recommended changes to the text have not been made

Reply:
Updated